### PR TITLE
endure has a higher cd

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Endure CD increased to an effective 50 seconds of cooldown

## Why It's Good For The Game
Endure makes ravager way too safe, and fifty seconds of effective cooldown isn't really that long in TGMC, and I doubt you can be caught off guard that often, I think the hardstats on endure are perfectly fine but the CD should be higher.

## Changelog
:cl:
balance: Endure has 60 second cooldown, effective 50
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm [hunk 1]
```diff
@@ -146,8 +146,8 @@
 	action_icon_state = "ignore_pain"
 	mechanics_text = "For the next few moments you will not go into crit and become resistant to explosives and immune to stagger and slowdown, but you still die if you take damage exceeding your crit health."
 	ability_name = "Endure"
-	plasma_cost = 200
-	cooldown_timer = 30 SECONDS
+	plasma_cost = 325
+	cooldown_timer = 60 SECONDS
```
Comment (Pariah919) [inline code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm:150]:
```suggestion
	cooldown_timer = 45 SECONDS
```

**Comment by Surrealaser** [comment]
Bad.

Even Ignore Pain didn't have anywhere close to 325 plasma cost and it was literal immortality.

As mentioned in #balance, I'd like to see how the old 40 second cooldown timer pans out, but 325, a whopping more than 50% plasma increase is inane.

File: code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm [hunk 2]
```diff
@@ -146,8 +146,8 @@
 	action_icon_state = "ignore_pain"
 	mechanics_text = "For the next few moments you will not go into crit and become resistant to explosives and immune to stagger and slowdown, but you still die if you take damage exceeding your crit health."
 	ability_name = "Endure"
-	plasma_cost = 200
-	cooldown_timer = 30 SECONDS
+	plasma_cost = 325
```
Comment (Pariah919) [inline code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm:149]:
```suggestion
	plasma_cost = 200
```

File: code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm [hunk 3]
```diff
@@ -146,8 +146,8 @@
...
+	cooldown_timer = 45 SECONDS
```
Comment (Pariah919) [inline code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm:150]:
```suggestion
	cooldown_timer = 60 SECONDS
```

**Review by Pariah919 — COMMENTED** [review]
splinter

**Comment by MLGTASTICa** [comment]
For a xeno that has a severe lack of long-ranged abilities  and is all focused around going rip and tear , the cooldown should not be increased , since endure is the only ability that lets him go and tear at groups of more than 3 marines without  being hit by a sniper and made to walk at snails pace , effectively killing them in most engagements.

**Pull request merged**